testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="x86 x86_64"
+license="GPL"
+depends="ftgl libxml2 sdl_image sdl_mixer"
+makedepends="sdl_image-dev sdl_mixer-dev ftgl-dev libxml2-dev"
+subpackages="$pkgname-doc"
+options="!check"
+source="https://launchpad.net/$pkgname/0.2.8/$pkgver/+download/$pkgname-$pkgver.src.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+cd "$builddir"
```

Comment:
No need to cd into builddir, abuild does that automatically.

Reply:
ok